### PR TITLE
Feature/testing

### PR DESCRIPTION
**Pull request by joshuacox**

### CODE REVIEW

File: addus [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    echo 'ssh dir exists, proceeding...'
+else
+    echo 'ssh dir does not exist, please generate it with "ssh-keygen -t ecdsa" for example'
+    ssh-keygen -b 521 -N "" -t ecdsa -f $HOME/.ssh/id_ecdsa
+fi
+echo 'This is for Webhosting.coop dev only, this will add our keys to your server, BEWARE!'
+touch $HOME/.ssh/authorized_keys
+TMP=$(mktemp -d --suffix=SDX)
+cp $HOME/.ssh/authorized_keys $TMP/mysshauthorized_keys
+curl --silent https://raw.githubusercontent.com/WebHostingCoopTeam/keys/master/keys |xargs -n1 -I{} echo '{}'>> $TMP/mysshauthorized_keys
```
Comment (eolszewski) [inline addus]: Should we have this pointing at out own directory or is this here as we don't currently have such a directory?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Transmute.industries pubs
+
+oneliner to authorize Transmute team  to a server
+```
+curl -Ls git.io/tstaddus|bash
```
Comment (eolszewski) [inline README.md:5]: Not as familiar with bash, but when I run this command I get `bash: line 1: 404:: command not found`.

**Review by eolszewski — COMMENTED** [review]
Waiting on a few questions before merging.

File: addus (hunk 1 above)
Comment (joshuacox) [reply addus]: This is a valid mishap, I thought I had replaced that line.   I may have reverted that file errantly.   Correcting shortly.

File: README.md (hunk 2 above)
Comment (joshuacox) [reply README.md:5]: this is a redirect to:
```
Location: https://raw.githubusercontent.com/transmute-industries/pubs/master/addus
```
and will not exist till this pull request goes through

**Comment by joshuacox** [comment]
looking at this again, I added a trap on the individual test to go ahead and rm the tmp even if the test fails, but I guess since this is only going to be leaving a tiny tmp file on a vm that is about  to be deleted in travis' cloud its not the biggest deal.

**Review by eolszewski — DISMISSED** [review]

**Comment by eolszewski** [comment]
@joshuacox looks like this needs to be committed with -S for it to be valid, is that right? I have not ever before signed commits with GPG, but it seems like that is the requirement here.

**Comment by joshuacox** [comment]
yep, lemme squash and sign and reopen as a new PR

**Pull request closed without merging**